project4/index.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://use.typekit.net/dtv5spx.css">
+    <title>#womendesigningwhileblack</title>
+</head>
+
+<body id="top">
+
+    <header>
+
+        <a style="font-size:1.75vw;" class="cta-tickets" href="#">tickets</a>
+
+        <img class="logo" src="images/wdwb-logo.svg" alt="WomenDesigningWhileBlack logo">
+
+        <div>
+            <h1 style="font-size:4vw;" class="title">#WomenDesigningWhileBlack</h1>
+            <h3 style="font-size:2.75vw;" class="theme">2020 - Confidence & Empowerment</h3>
+        </div>
+
+    </header>
+
+    <nav>
+        <ul>
+            <!-- <li><a href="#home"></a></li> -->
+            <li><a href="#about">about</a></li>
+            <li><a href="#2020">2020</a></li>
+            <li><a href="#attend">attend</a></li>
+            <li><a href="#speakers">speakers</a></li>
+            <li class="special"><a href="#exhibitions-learning">exhibitions & learning</a></li>
+            <li><a href="#schedule">schedule</a></li>
+        </ul>
+    </nav>
+
+    <div class="main-image">
```

Comment:
Now that I see you have a container around this image and given it a `hidden` `overflow-x` value, I think you _can_ do your oversized image idea!

Reply:
I added that in after we talked in class, but it was still allowing the horizontal scroll, so I must not have entered it correctly? I'm not sure.